common/src/main/scala/explore/common/ConstraintSetObsQueries.scala

```diff
@@ -110,7 +109,7 @@ object ConstraintSetObsQueries {
         LiveQueryRenderMod[ObservationDB, ConstraintSetsObsQuery.Data, ConstraintSetsWithObs](
           ConstraintSetsObsQuery.query(),
           ConstraintSetsObsQuery.Data.asConstraintSetsWithObs.get,
-          NonEmptyList.of(
+          List(
```

Comment:
I enabled having zero subscriptions indicating refreshes. This works effectively as a single shot query.